sources/packages/api/src/route-controllers/BaseController.ts

```diff
@@ -0,0 +1,5 @@
+export default class BaseController {
+  public handleRequestError(e: any) {
+    console.log(e);
```

Comment:
Re-throw error, nest error mechanism will handle this end send 502 error, otherwise in error condition server become unresponsive.

Reply:
Its already thrown in the caller  i.e. user.controller.ts line 29.